lib/school.rb

```diff
@@ -0,0 +1,21 @@
+class School
+  attr_reader :start_time, :hours_in_school_day
+
+  def initialize(start_time, hours_in_school_day)
+    @start_time          = start_time
+    @hours_in_school_day = hours_in_school_day
+  end
+
+  def student_names
```

Comment:
`student_names` should be an intance variable set to an empty array in your `initialize` method